### PR TITLE
Remove redundant libraries linked into swupd binary

### PR DESCRIPTION
**Pull request by matthewrsj**
Signed-off-by: Matthew Johnson <matthew.johnson@intel.com>

### CODE REVIEW

File: docs/swupd.1 [hunk 1]
```diff
@@ -400,8 +400,8 @@ Examples:
 Ignores \fB/var\fP or \fB/etc/machine\-id\fP, regardless of
 whether they are directories or something else. In the
 usual case that \fB/var\fP is a directory, also everything
-inside it is ignored because directories get skipped
-during scanning the directory tree.
+inside it is ignored because the directory gets skipped
+while scanning the directory tree.
```
Comment (phmccarty) [inline docs/swupd.1]: Unrelated change? I would separate this into a separate commit if it's a valid change.

Comment (matthewrsj) [reply docs/swupd.1]: sure.

Comment (matthewrsj) [reply docs/swupd.1]: Moved to own commit.
